src/en/help-oracle.md

```diff
@@ -2,94 +2,94 @@ Title: Using the Oracle cloud
 
 # Using the Oracle public cloud
 
-Juju has built-in support for [Oracle's Compute service][compute]. However, the
-following few steps are required before Juju can bootstrap Oracle's Compute
-cloud:
+Juju has built-in support for [Oracle Compute][oracle-compute], Oracle's public
+cloud. This means that there is no need to add the Oracle cloud to Juju. An
+exception to this are Oracle Compute trial accounts. Both types of accounts,
+paid and trial, are covered here.
 
-1. Associate Oracle's Ubuntu images with your Compute service. Juju uses these
-   for deployment.
+!!! Warning:
+    Support for Oracle Compute is only available via the [juju 2.2 dev
+    release][jujubeta] and is therefore not yet recommended for production use.
 
-1. For trial accounts, add the Oracle cloud to Juju. Endpoint details may need 
-   to be retrieved from the Oracle Compute *Dashboard*.
-   Juju already knows about endpoints for paid accounts so this step is only
-   necessary if using a trial account.
+The email you received upon signing up for Oracle Compute contains vital
+information you will need to complete the instructions presented here. Look
+for:
 
-1. Add authentication credentials for the specific Compute service to Juju.
+- 'My Services URL'
+- 'Identity domain'
+- 'My Account URL'
+- 'Username' (your email address)
+- 'Password' (temporary)
 
-We'll step through each of these requirements below, before using Juju to
-launch a test deployment on Oracle's Compute. 
+Upon first login (using either of the above URLs) you will be prompted to
+change the temporary password to arrive at your final password.
 
-!!! Note:
-	Oracle support is currently in the latest [Juju 2.2 release
-        candidate][jujubeta] and is experimental. Both the Juju and Oracle
-        teams are working to improve the user experience.
+## Ubuntu images
 
-## Images
+You will need to make Ubuntu images available in your Oracle Compute account in
+order for Juju to be able to create Ubuntu-based machines. This is a
+requirement.
 
-Juju needs access to Ubuntu images that have been associated with your Oracle
-Compute deployment. 
+Proceed as follows:
 
-To do this, first sign in to your Oracle cloud domain. The URL for this will
-depend on the cloud's geographical location. In EMEA, for example, the URL is
-the following:
+1. Navigate to 'My Services URL' and log in.
+1. Select your 'Identity Domain' in the top-right corner.
```

Comment:
Yes, the user may have more than one. The screenshot I included shows 'Identity Domain':
![oracle_empty-dashboard-2](https://user-images.githubusercontent.com/6303808/28326532-ffcc65e8-6bae-11e7-80ca-188513fd0b71.png)